interop/grpc_testing/test.proto

```diff
@@ -214,6 +214,13 @@ message LoadBalancerStatsRequest {
 }
 
 message LoadBalancerStatsResponse {
+  message RPCsByPeer {
+    // The number of completed RPCs for each peer.
+    map<string, int32> rpcs_by_peer = 1;
+  }
+  // The number of completed RPCs for each type (UnaryCall or EmptyCall).
+  map<string, RPCsByPeer> rpcs_by_type = 3;
```

Comment:
Nit: These are out of order now (3 is before 1 and 2). Move this block to after `num_failures`?

Reply:
Done. Also fixed in https://github.com/grpc/grpc/pull/23436